yoyodyne/models/base.py

```diff
@@ -339,61 +339,17 @@ def _get_lr_scheduler(
         return [scheduler_cfg]
 
     def _get_loss_func(
-        self, reduction: str
```

Comment:
I think I agree that there is not much reason to expose this, but I just want to make sure this is intentional (one could hypothetically want to sum losses).

Reply:
Yep, it is just me taking deliberate advantage of the default behavior since we never use any of the alternatives.